### ISSUE TITLE
PathOGiST build is failing

### ISSUE THREAD

**Issue by realseanla** [WGS-TB/PathOGiST#31]
The Travis CI build is failing for master branch. @JuliusBooth and @matnguyen, your commits seem to be the first commits to fail after a string of successfully built commits. Could you guys look into that? My commits were only modifying the README.

**Comment by matnguyen** [comment]
```
Traceback (most recent call last):
  File "./PATHOGIST", line 863, in <module>
    main()
  File "./PATHOGIST", line 852, in main
    run_all(param, MAJOR_VERSION, MINOR_VERSION, PATCH_VERSION)
  File "./PATHOGIST", line 640, in run_all
    call_clustering_commands(clustering_args,run_args,denovo_calls_dists_paths,threads,temp_dir)
  File "./PATHOGIST", line 517, in call_clustering_commands
    calls[genotype] = read_genotyping_calls(genotype,denovo_calls_paths[genotype],clustering_args)
  File "./PATHOGIST", line 81, in read_genotyping_calls
    return read_calls_functions[genotype](calls_path)
  File "/home/travis/build/WGS-TB/PathOGiST/pathogist/io.py", line 50, in read_mlst_calls
    with open(calls_path,'r') as calls_file:
FileNotFoundError: [Errno 2] No such file or directory: 'tests/integration_tests/temp_dir/SRR6153024_mlst.call'
/home/travis/.travis/functions: line 524:  7406 Terminated              travis_jigger "${!}" "${timeout}" "${cmd[@]}"
The command "travis_wait 180 tests/integration_tests/integration_test.sh" exited with 1.
```

Has there been any modifications to the integrations tests data? I haven't changed anything there 

CPLEX also fails:
```
ERROR: test_correlation (tests.unit_tests.test_cluster.ClusterTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/WGS-TB/PathOGiST/tests/unit_tests/test_cluster.py", line 35, in test_correlation
    clustering = pathogist.cluster.correlation(self.mlst_dist,500)
  File "/home/travis/build/WGS-TB/PathOGiST/pathogist/cluster.py", line 536, in correlation
    sol_matrix = processProblem(weight_matrix.values, all_constraints, start_solution)
  File "/home/travis/build/WGS-TB/PathOGiST/pathogist/cluster.py", line 108, in processProblem
    my_prob = cplex.Cplex()
NameError: name 'cplex' is not defined
```

**Comment by JuliusBooth** [comment]
Must be from changes Johnathan made on April 12th/13th. File error maybe from commit 7963a3b

**Comment by jowong4** [comment]
`11:15:16 PM (95077 ms) -> INFO:Finished constructing database.

11:15:16 PM (95077 ms) -> INFO: Calling MLSTs on samples SRR6152717 using MentaLiST...

ERROR: could not open file /home/travis/anaconda3/envs/pathogist/bin/julia_mentalist/MentaLiST-65451e75e5cfd168c23989a0ef50fcf545cc8e57/src/MentaLiST.jl

Stacktrace:

 [1] include at ./boot.jl:326 [inlined]

 [2] include_relative(::Module, ::String) at ./loading.jl:1038

 [3] include(::Module, ::String) at ./sysimg.jl:29

 [4] exec_options(::Base.JLOptions) at ./client.jl:267

 [5] _start() at ./client.jl:436

11:15:17 PM (96368 ms) -> INFO: Finished calling MLSTs on sample SRR6152717.

11:15:17 PM (96368 ms) -> INFO: Calling MLSTs on samples SRR6153024 using MentaLiST...

ERROR: could not open file /home/travis/anaconda3/envs/pathogist/bin/julia_mentalist/MentaLiST-65451e75e5cfd168c23989a0ef50fcf545cc8e57/src/MentaLiST.jl` I think this is the issue. There must have been some problem with installing mentalist. Do we know if the conda version of mentalist is stable?

**Comment by realseanla** [comment]
Oh yeah, seems there is an error with the MentaLiST build. I'll make an issue with them.

**Comment by jowong4** [comment]
It's probably not them, but when I was working on including the genotyping tools, mentalist on conda wasnt working so I had to manually install Ruby and mentalist from GitHub. I'll take another look at the error code when I get home and see if it is on my end. And if there is a stable mentalist on conda, we can install that instead

**Comment by realseanla** [comment]
Okay, thanks so much @jowong4!

**Comment by jowong4** [comment]
I seem to recall packages were at least one of the issues I encountered which is why I resorted to manually installing mentalist and it's dependencies

**Comment by jowong4** [comment]
Both fail builds and cplex errors are resolved in #32 . See pull request more details. @realseanla , I do suggest getting mentalist working with conda so we don't have to use this hack
